os/net/lwip/src/include/lwip/netifapi.h

```diff
@@ -85,6 +87,11 @@ struct netifapi_msg {
 			netifapi_void_fn voidfunc;
 			netifapi_errt_fn errtfunc;
 		} common;
+    struct {
+      netifapi_void_fn_arg voidfunc;
+			netifapi_errt_fn_arg errtfunc;
```

Comment:
can we do indentation for this structure

Reply:
Thanks I'll modify it